src/spikeinterface/postprocessing/template_similarity.py

```diff
@@ -234,8 +234,8 @@ def compute_similarity_with_templates_array(
                     distances[count, i, j] = sklearn.metrics.pairwise.pairwise_distances(src, tgt, metric="l2")
                     distances[count, i, j] /= norm_i + norm_j
                 else:
-                    distances[count, i, j] = sklearn.metrics.pairwise.pairwise_distances(src, tgt, metric="cosine")
-
+                    distance = sklearn.metrics.pairwise.pairwise_distances(src, tgt, metric="cosine")
+                    distances[count, i, j] = distance[0, 0]
```

Comment:
@rkim48 can you move this to a different PR? Not sure if this is a matter of the sklearn version